src/Geo/AABB2D.php

```diff
@@ -70,7 +70,7 @@ public function copy() : AABB2D
     /**
      * Returns the center of the AABB
      */
-    public function getCenter() : Vec2
+    public function getCenter() : float|Vec2
```

Comment:
Same here

Reply:
it's not only float, it's float or Vec3

Reply:
Yes, but that is incorrect, look at the source: https://github.com/mario-deluna/php-glfw/blob/master/phpglfw_math.c#L317

This is the implementation of the `getCenter`function:

```
($this->min + $this->max) * 0.5;
```

aka 

```
(Vec2 + Vec2) * 0.5 
```

This expression will **always** return a `Vec2` and **never** a scalar `float`.

Reply:
So, should I remove `float` return types, then add these to `phpstan.neon` as ignored errors?